Testing/functional/e2e/cypress/integration/services/medication.js

```diff
@@ -56,25 +104,25 @@ describe("Medication Service", () => {
 
     it("Verify MedicationStatement Authorized", () => {
         const HDID = "P6FFO433A5WPMVTGM7T4ZVWBKCSVNAYGTWTU3J2LWMGUMERKI72A";
-        cy.fixture("MedicationService/medicationStatement.json").then((MedicationStatementResponse) => {
```

Comment:
this fixture needs to be deleted if its not used anymore

Reply:
removed the json